src/state-client.ts

```diff
@@ -218,6 +219,10 @@ class StateClient {
     this.token = token;
   }
 
+  public sendTransactionToServer(identifier: string, tx: Transaction) {
+    this.signedRequest<Transaction, Record<string, unknown>>(identifier, "POST", "bits/transaction", tx);
+  }
+
```

Comment:
Can you move this down to the bottom of this file (with the other simple wrappers) and add an `@ignore` comment like the others?

Reply:
The endpoint is actually `bits/transactions` (plural). I think I typed that wrong in Slack.